material-overrides/assets/stylesheets/polkadot.css

```
@@ -1371,16 +1374,31 @@ details.interface > summary::before {
 
 /** Feedback widget styling **/
 .md-feedback {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   width: fit-content;
-  padding: 1em;
-  border: 1px solid var(--mystic);
-  border-radius: 16px;
-  margin: 1rem auto;
+  padding: 0;
+  margin-top: 0px;
+  margin-bottom: 0px;
```

```suggestion
  margin-top: 0;
  margin-bottom: 0;
```

For consistency with the rest of the file, if you set something to `0px`, please just set it to `0` instead